### PR TITLE
Problem: no command to start a sandbox environment

### PR DESCRIPTION
**Pull request by farazsth98**
PR for Issue https://github.com/crypto-org-chain/gravity-bridge/issues/189

I updated the `readme.md` a little bit here as well to reflect the actual integration test setup on this repository (which is different from the original gravity-bridge repo). Feel free to make any other changes before approving the PR :)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -68,23 +68,43 @@ Currently on every commit we send hundreds of transactions, dozens of validator
...
+
+### Running the chains as a playground
+
+There is a "sandbox" style test, which simply brings up the chain and allows you as the developer to perform whatever operations you want. 
+
+To interact with the ethereum side of the chain, use the JSON-RPC server located at `http://localhost:8545`. You may choose to use `foundry` or `hardhat` to make this easier.
+
+To interact with the gravity module itself, you'll want to build the `gravity` module locally by running the following command.
+
+`user@pc:/gravity-bridge/module$ go build -o build/gravity cmd/gravity/main.go`
```
Comment (thomas-nguy) [inline readme.md]:
```suggestion
`
cd module
make build
`
```

File: readme.md [hunk 2]
```diff
@@ -68,23 +68,43 @@ Currently on every commit we send hundreds of transactions, dozens of validator
...
+
+You can then use this binary to interact with the Cosmos side of the chain. Check out the Cosmos documentation for more inforatmion on this.
```
Comment (thomas-nguy) [inline readme.md]:
```suggestion
You can then use the binary in the build directory to interact with the Cosmos side of the chain. Check out the Cosmos documentation for more information on this.
```

**Comment by thomas-nguy** [comment]
minor suggestions on the readme, otherwise lgtm

**Review by thomas-nguy — DISMISSED** [review]

**Comment by thomas-nguy** [comment]
Is it possible to have your commits signed as it is a requirement for merging

**Comment by farazsth98** [comment]
@thomas-nguy sure, I'm not 100% how to do that so let me figure that out and I'll get it done asap

**Comment by farazsth98** [comment]
Are the test failures related to the unsigned commits? @thomas-nguy

**Comment by thomas-nguy** [comment]
no, the tests failure is caused by permission issue (cannot publish artifact from forked repo) but I think those changes won't affect the integration tests so it can be ignored

**Comment by farazsth98** [comment]
@thomas-nguy had to force push after I figured out how to sign my old commits, but is it good now? I see it says "Verified" next to the commits

**Review by thomas-nguy — APPROVED** [review]

**Comment by thomas-nguy** [comment]
looks good, thank you 👍

**Pull request merged**